### PR TITLE
[ZH] Add Beatmap Spotlights Spring 2022 playlist B

### PR DESCRIPTION
**Pull request by Walavouchey**
these weren't copied over in https://github.com/ppy/osu-wiki/pull/7454

also the "suggested by" isn't translated (cc @Kotoki1337, @heipizhu4)

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] translate "suggested by"

### CODE REVIEW

File: wiki/Beatmap_Spotlights/Seasons/2022_Spring/zh.md [hunk 1]
```diff
@@ -55,6 +53,18 @@ tags:
 - [Iyowa feat. KAFU - Kyu-kurarin (0ugi) \[Anxiety\]](https://osu.ppy.sh/beatmapsets/1591460#osu/3250375), suggested by [DeviousPanda](https://osu.ppy.sh/users/4966334)
 - [Yousei Teikoku - Zetsu (Mazzerin) \[Mazzerin's Extreme\]](https://osu.ppy.sh/beatmapsets/1262446#osu/2929651), suggested by [Omgforz](https://osu.ppy.sh/users/578943)
 
+#### 歌单 B
+
+- [Annabel - little voice (Niva) \[Niva's Hard\]](https://osu.ppy.sh/beatmapsets/1619033#osu/3343181), suggested by [Nowaie](https://osu.ppy.sh/users/5428909)
```
Comment (Kotoki1337) [inline wiki/Beatmap_Spotlights/Seasons/2022_Spring/zh.md:58]: translation format: 

```suggestion
- [Annabel - little voice (Niva) \[Niva's Hard\]](https://osu.ppy.sh/beatmapsets/1619033#osu/3343181)，由 [Nowaie](https://osu.ppy.sh/users/5428909) 推荐
```

**Review by Kotoki1337 — CHANGES_REQUESTED** [review]
my bad, i forgot to translate these things. should i make a pr to your fork or you just deal them with suggestion?

**Comment by Walavouchey** [comment]
Ty, applied

File: wiki/Beatmap_Spotlights/Seasons/2022_Spring/zh.md [hunk 2]
```diff
@@ -45,54 +43,102 @@ tags:
...
-- [\[Kyoro\] - Mochio- (Snow Note) \[Insane\]](https://osu.ppy.sh/beatmapsets/1473006#osu/3028465), suggested by [Riot](https://osu.ppy.sh/users/4256461)
-- [ELFENSJoN - Naraku no Majo (Woood13) \[Woood's Insane\]](https://osu.ppy.sh/beatmapsets/1671498#osu/3457798), suggested by [Lefafel](https://osu.ppy.sh/users/2295850)
-- [FELT - RESISTANCE (Kalibe) \[ENDURANCE\]](https://osu.ppy.sh/beatmapsets/1004248#osu/2101856), suggested by [Omgforz](https://osu.ppy.sh/users/578943)
-- [KikuoHana - Hikari yo (SMOKELIND) \[EX\]](https://osu.ppy.sh/beatmapsets/1524556#osu/3119298), suggested by [Riot](https://osu.ppy.sh/users/4256461)
-- [Iyowa feat. KAFU - Kyu-kurarin (0ugi) \[Anxiety\]](https://osu.ppy.sh/beatmapsets/1591460#osu/3250375), suggested by [DeviousPanda](https://osu.ppy.sh/users/4966334)
-- [Yousei Teikoku - Zetsu (Mazzerin) \[Mazzerin's Extreme\]](https://osu.ppy.sh/beatmapsets/1262446#osu/2929651), suggested by [Omgforz](https://osu.ppy.sh/users/578943)
+- [Otsukisama Koukyoukyoku - ultima Thule (Kyle Y) \[Ky's Hard\]](https://osu.ppy.sh/beatmapsets/1507229#osu/3204358), 由 [Nowaie](https://osu.ppy.sh/users/5428909) 推荐
```
Comment (Kotoki1337) [inline wiki/Beatmap_Spotlights/Seasons/2022_Spring/zh.md:46]: should using fullweight comma

```suggestion
- [Otsukisama Koukyoukyoku - ultima Thule (Kyle Y) \[Ky's Hard\]](https://osu.ppy.sh/beatmapsets/1507229#osu/3204358)，由 [Nowaie](https://osu.ppy.sh/users/5428909) 推荐
```

**Review by Kotoki1337 — CHANGES_REQUESTED** [review]

File: wiki/Beatmap_Spotlights/Seasons/2022_Spring/zh.md (hunk 2 above)
Comment (Walavouchey) [reply wiki/Beatmap_Spotlights/Seasons/2022_Spring/zh.md]: woops, applied incorrectly

File: wiki/Beatmap_Spotlights/Seasons/2022_Spring/zh.md [hunk 3]
```diff
@@ -45,54 +43,102 @@ tags:
...
+- [FELT - RESISTANCE (Kalibe) \[ENDURANCE\]](https://osu.ppy.sh/beatmapsets/1004248#osu/2101856)，由 [Omgforz](https://osu.ppy.sh/users/578943) 推荐
+- [KikuoHana - Hikari yo (SMOKELIND) \[EX\]](https://osu.ppy.sh/beatmapsets/1524556#osu/3119298)，由 [Riot](https://osu.ppy.sh/users/4256461) 推荐
+- [Iyowa feat. KAFU - Kyu-kurarin (0ugi) \[Anxiety\]](https://osu.ppy.sh/beatmapsets/1591460#osu/3250375)，由 [DeviousPanda](https://osu.ppy.sh/users/4966334) 推荐
+- [Yousei Teikoku - Zetsu (Mazzerin) \[Mazzerin's Extreme\]](https://osu.ppy.sh/beatmapsets/1262446#osu/2929651)，由 [Omgforz](https://osu.ppy.sh/users/578943) 推荐
+
+#### 歌单 B
+
+- [Annabel - little voice (Niva) \[Niva's Hard\]](https://osu.ppy.sh/beatmapsets/1619033#osu/3343181)，由 [Nowaie](https://osu.ppy.sh/users/5428909) 推荐
+- [Silver Forest - Kero (9) destiny (Bloxi) \[Hard\]](https://osu.ppy.sh/beatmapsets/1670776#osu/3423825)，由 [DeviousPanda](https://osu.ppy.sh/users/4966334) 推荐
+- [Aimer - Kataomoi (Moecho) \[Darlin'，Aishiteru.\]](https://osu.ppy.sh/beatmapsets/1058659#osu/2217225)，由 [Zelq](https://osu.ppy.sh/users/8953955) 推荐
```
Comment (Kotoki1337) [inline wiki/Beatmap_Spotlights/Seasons/2022_Spring/zh.md:60]:
```suggestion
- [Aimer - Kataomoi (Moecho) \[Darlin', Aishiteru.\]](https://osu.ppy.sh/beatmapsets/1058659#osu/2217225)，由 [Zelq](https://osu.ppy.sh/users/8953955) 推荐
```

File: wiki/Beatmap_Spotlights/Seasons/2022_Spring/zh.md [hunk 4]
```diff
@@ -45,54 +43,102 @@ tags:
...
-- [Frums - VIS::CRACKED (\[Crz\]Crysarlene) \[Mipha vs. Crys's Oblivion\]](https://osu.ppy.sh/beatmapsets/1561114#mania/3188294), suggested by [Crumpey](https://osu.ppy.sh/users/3518705)
+- [DJ TOTTO - Crystalia (Critical_Star) \[CS' Hyper\]](https://osu.ppy.sh/beatmapsets/782749#mania/1643587)，由 [Crumpey](https://osu.ppy.sh/users/3518705) 推荐
+- [katagiri - HIVEMIND (DannyPX) \[Hyper\]](https://osu.ppy.sh/beatmapsets/1665442#mania/3411830)，由 [Crumpey](https://osu.ppy.sh/users/3518705) 推荐
+- [Umeboshi Chazuke - Bison Charge (\_Asha) \[Insane\]](https://osu.ppy.sh/beatmapsets/838465#mania/1785223)，由 [Crumpey](https://osu.ppy.sh/users/3518705) 推荐
+- [Five Hammer - fffff op.2 (biemote) \[bie's Insane\]](https://osu.ppy.sh/beatmapsets/1452883#mania/2987140)，由 [lenpai](https://osu.ppy.sh/users/5314573) 推荐
+- [Frums - Star of the COME ON!! (Kawawa) \[INSANE!!\]](https://osu.ppy.sh/beatmapsets/1305627#mania/2706983)，由 [lenpai](https://osu.ppy.sh/users/5314573) 推荐
+- [Camellia - \* Erm，could it be a Spatiotemporal ShockWAVE Syndrome...? (Dubstek) \[YABAI\]](https://osu.ppy.sh/beatmapsets/1477789#mania/3031881)，由 [Crumpey](https://osu.ppy.sh/users/3518705) 推荐
```
Comment (Kotoki1337) [inline wiki/Beatmap_Spotlights/Seasons/2022_Spring/zh.md:129]:
```suggestion
- [Camellia - \* Erm, could it be a Spatiotemporal ShockWAVE Syndrome...? (Dubstek) \[YABAI\]](https://osu.ppy.sh/beatmapsets/1477789#mania/3031881)，由 [Crumpey](https://osu.ppy.sh/users/3518705) 推荐
```

**Review by Kotoki1337 — CHANGES_REQUESTED** [review]
`Ctrl + F` moment 💀

**Review by Kotoki1337 — APPROVED** [review]
perfect 👍

**Review by TicClick — APPROVED** [review]

**Pull request merged**